data/moves.js

```diff
@@ -11142,7 +11142,7 @@ exports.BattleMovedex = {
 		name: "Rain Dance",
 		pp: 5,
 		priority: 0,
-		flags: {},
+		flags: {dance: 1},
```

Comment:
Appears that all of these moves except Rain Dance have Dance in their JAP names. Thanks for pointing out xfix!